From 69a393d260737ae3dbd89fe3127e8954f7a285c9 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Thu, 28 Jul 2016 16:47:49 +0800
Subject: [PATCH] ARM64: configs: rockchip_cros_defconfig: enable EAS

This patch trys to sync the rockchip_defconfig for cros.

Change-Id: Ieeda1e41f05f33ffaa708c9825901553a759b17b
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/configs/rockchip_cros_defconfig | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/rockchip_cros_defconfig b/arch/arm64/configs/rockchip_cros_defconfig
index 5f8f7ba6efef..f36e67727b3c 100644
--- a/arch/arm64/configs/rockchip_cros_defconfig
+++ b/arch/arm64/configs/rockchip_cros_defconfig
@@ -21,6 +21,7 @@ CONFIG_RT_GROUP_SCHED=y
 CONFIG_NAMESPACES=y
 # CONFIG_UTS_NS is not set
 # CONFIG_PID_NS is not set
+CONFIG_SCHED_TUNE=y
 CONFIG_RELAY=y
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
@@ -68,11 +69,12 @@ CONFIG_WQ_POWER_EFFICIENT_DEFAULT=y
 CONFIG_CPU_IDLE=y
 CONFIG_ARM_CPUIDLE=y
 CONFIG_CPU_FREQ=y
-CONFIG_CPU_FREQ_DEFAULT_GOV_INTERACTIVE=y
+CONFIG_CPU_FREQ_DEFAULT_GOV_SCHED=y
 CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_USERSPACE=y
 CONFIG_CPU_FREQ_GOV_ONDEMAND=y
+CONFIG_CPU_FREQ_GOV_INTERACTIVE=y
 CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
 CONFIG_CPUFREQ_DT=y
 CONFIG_ARM_ROCKCHIP_CPUFREQ=y
@@ -210,7 +212,6 @@ CONFIG_ZRAM_LZ4_COMPRESS=y
 CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_RAM=y
 CONFIG_BLK_DEV_RAM_SIZE=8192
-CONFIG_UID_STAT=y
 CONFIG_SRAM=y
 CONFIG_ROCKCHIP_SCR=y
 CONFIG_SCSI=y
-- 
2.35.3

